cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -210,3 +211,39 @@ Then("The whole row is highlighted", () => {
 Then("Flat table caption is set to {word}", (text) => {
   flatTableCaption().should("have.text", text);
 });
+
+When("I click on the {word} cell in the first row", (position) => {
+  flatTableCell(positionOfElement(position)).click();
+});
+
+Then("The subrows are visible", () => {
+  flatTableSubRow().should("be.visible");
```

Comment:
subrow**S** or subrow? because we are calling cy.get('[data-element="flat-table-sub-row"]')?